save-orchestrator/src/main/resources/application-mac.properties

```diff
@@ -1,2 +1,3 @@
 orchestrator.executionLogs=/Users/Shared/cnb/executionLogs/
 orchestrator.testResources.basePath=/Users/Shared/cnb/repositories
+orchestrator.adjustResourceOwner=false
```

Comment:
What is this for?

Reply:
This settings controls whether orchestrator `chown`s resources when packaging them into a container. I'm also curious why it should be disabled on Mac, maybe permission control differs from Linux?